sys/net/routing/nhdp/nhdp.h

```diff
@@ -45,6 +45,9 @@ extern "C" {
 #define MANET_PORT                  (269)
 #endif
 
+/** @brief Maximum number of registerable interfaces */
+#define NHDP_MAX_REG_MANET_IFS      (1)
```

Comment:
That's one of the reasons why I would recommend an external define: that's easier to replace when refactoring. But now that I think about it a little more, maybe it's indeed better to use directly `NG_NETIF_NUMOF` since the header is already merged.